plugins/janus_auviousrecordplay.c

```diff
@@ -1903,8 +1910,8 @@ static void *janus_auviousrecordplay_handler(void *data) {
 			json_decref(jsep);
 		}
 		janus_auviousrecordplay_message_free(msg);
+        JANUS_LOG(LOG_INFO, "  >> Listed recordings: %u\n", g_hash_table_size(recordings));
```

Comment:
just indent, and maybe set to LOG_VERB?

Reply:
The log was purposely left to LOG_INFO until tests are finished, even though g_hash_table_size  is a low-cost operation.

The indent on my vs code looks fine must be formatting character misalignment somewhere. 

I tested thoroughly - would like to deploy it to dev for monitoring on GKE over the weekend before closing this issue.